stockflux-securities-master/src/components/Message.css

```diff
@@ -0,0 +1,9 @@
+.securities-success-message {
```

Comment:
Maybe you could have a `.securities-message` class as the based and then allow for adding `.error` or `.success` as modifiers which only change the colour. Just like we did before.

Reply:
yep, good point, I've changed it now